src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -119,9 +136,14 @@ def __init__(
         dt=None,
         dt_range=None,
         field_weighting="volume averaged",
+        integrator="explicit_boris_relativistic",
```

Comment:
I think it would be better if the tracker just switched over automatically if the simulation was relativistic? We have the velocities, and there's no circumstance where someone would want to use the wrong integrator?

Reply:
According to the profiler library they both have a percall time of 0.000 so I think it's safe to say they're close enough? What do you think